dynomitemanager/src/main/java/com/netflix/dynomitemanager/identity/DefaultVpcInstanceEnvIdentity.java

```diff
@@ -0,0 +1,22 @@
+package com.netflix.dynomitemanager.identity;
```

Comment:
@ipapapa 

The problem is one the AWSMembership here:
https://github.com/Netflix/dynomite-manager/blob/dev/dynomitemanager/src/main/java/com/netflix/dynomitemanager/sidecore/aws/AWSMembership.java#L172

IMHO Because in the end of the day the cluster name == ASG name for you guys. So It get the data from the Netflix_APP env var if there is nothing ther it uses: dynomite_demo1. 

You get the information from config.getACLGroupName which I don't have it! So the name filter was messed up. I just added a IF to check is there is a VpcInstanceDataRetriever being used and them I use the information from the Security Group doign this: 

SystemUtils.getDataFromUrl("http://169.254.169.254/latest/meta-data/network/interfaces/macs/" + mac + "/security-groups").trim();

And this code was added here VpcInstanceDataRetriever => That's why I added InstanceDataRetriever.String getSecurityGroupName(); and coded at: VpcInstanceDataRetriever.getSecurityGroupName()

So the code is:

Filter nameFilter = null;
if(instanceDataRetriever instanceof VpcInstanceDataRetriever){
	nameFilter = new Filter().withName("group-name").withValues(instanceDataRetriever.getSecurityGroupName()); //SG
}else{
	nameFilter = new Filter().withName("group-name").withValues(config.getACLGroupName()); //SG
}

I could start using this convetion IF you think is better / proper. IMHO It make sense read this from the AWS metadata like I did instead of adding mroe config in the CODE but thats your call :-)